doc/source/release/1.15.0-notes.rst

```diff
@@ -17,96 +17,1488 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+- `scipy.stats.wilcoxon` has improved method resolution logic for the default
+  ``method='auto'``. Other values of ``method`` provided by the user are now
+  respected in all cases, and the method argument ``approx`` has been
+  renamed to ``asymptotic`` for consistency with similar functions. (Use of
+  ``approx`` is still allowed for backward compatibility.)
+- There are several new probability distributions:
 
-*******************
-Deprecated features
-*******************
+  - `scipy.stats.dpareto_lognorm` represents the double Pareto lognormal
+    distribution.
+  - `scipy.stats.landau` represents the Landau distribution.
+  - `scipy.stats.normal_inverse_gamma` represents the normal-inverse-gamma
+    distribution.
+  - `scipy.stats.poisson_binom` represents the Poisson binomial distribution.
 
-`scipy.linalg` deprecations
-===========================
+- Batch calculation with `scipy.stats.alexandergovern` and
+  `scipy.stats.combine_pvalues` is faster.
+- `scipy.stats.chisquare` added an argument ``sum_check``. By default, the
+  function raises an error when the sum of expected and obseved frequencies
+  are not equal; setting ``sum_check=False`` disables this check to
+  facilitate hypothesis tests other than Pearson's chi-squared test.
+- The accuracy of several distribution methods has been improved, including:
 
+  - `scipy.stats.nct` method ``pdf``
+  - `scipy.stats.crystalball` method ``sf``
+  - `scipy.stats.geom` method ``rvs``
+  - `scipy.stats.cauchy` methods ``logpdf``, ``pdf``, ``ppf`` and ``isf``
+  - The ``logcdf`` and/or ``logsf`` methods of distributions that do not
+    override the generic implementation of these methods, including
+    `scipy.stats.beta`, `scipy.stats.betaprime`, `scipy.stats.cauchy`,
+    `scipy.stats.chi`, `scipy.stats.chi2`, `scipy.stats.exponweib`,
+    `scipy.stats.gamma`, `scipy.stats.gompertz`, `scipy.stats.halflogistic`,
+    `scipy.stats.hypsecant`, `scipy.stats.invgamma`, `scipy.stats.laplace`,
+    `scipy.stats.levy`, `scipy.stats.loggamma`, `scipy.stats.maxwell`,
+    `scipy.stats.nakagami`, and `scipy.stats.t`.
 
-`scipy.spatial` deprecations
-============================
+- `scipy.stats.qmc.PoissonDisk` now accepts lower and upper bounds
+  parameters ``l_bounds`` and ``u_bounds``.
+- `scipy.stats.fisher_exact` now supports two-dimensional tables with shapes
+  other than ``(2, 2)``.
+- `scipy.stats.truncate` adds the ability to truncate the distribution
+  underlying a random variable and `scipy.stats.abs` provides the ability to
+  take the absolute value of a random variable. The new ``FoldedDistribution``
+  represents the distribution underlying the absolute value of a random
+  variable.
+- Added ``OrderStatisticDistribution``, a class representing the distribution
+  of order statistics of samples drawn from another distribution.
+- It is now possible to generate quasi-random samples from statistical
+  distributions by providing a ``QMCEngine`` object to the ``rng`` argument.
```

Comment:
```suggestion
```
This is only part of the `sample` method of the new RV objects, so it's not really an enhancement that needs to be covered separately.